src/utils/streams.js

```diff
@@ -0,0 +1,158 @@
+import program from "commander"
+import readline from "readline"
+import reverseString from "reverse-string"
+import fs from "fs"
+import csv from "csvtojson"
+import Deque from "collections/deque";
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function reverse() {
+  rl.question("Please provide a string for reversing: ", (answer) => {
+    console.log(`Reversed: ${reverseString(answer)}`);
+    rl.close();
+  });
+}
+
+function transform() {
+  rl.question("Please provide a string for transforming to UPPER case: ", (answer) => {
+    console.log(`Transformed to upper case: ${answer.toUpperCase()}`);
+    rl.close();
+  });
+}
+
+function outputFile(filePath) {
+  validateFilePath(filePath, 'outputFile');
+  fs.createReadStream(filePath)
+    .on('error', handleFileReadError)
+    .pipe(process.stdout);
+}
+
+function convertFromFile(filePath) {
+  validateFilePath(filePath, 'convertFromFile');
+  fs.createReadStream(filePath)
+  .on('error', handleFileReadError)
+  .pipe(csv())
+  .pipe(process.stdout);
+}
+
+function convertToFile(filePath) {
+  validateFilePath(filePath, 'convertToFile');
+  fs.createReadStream(filePath)
+    .on('error', handleFileReadError)
+    .pipe(csv())
+    .pipe(fs.createWriteStream(filePath.replace(/(.*)\.csv$/, '$1.json')))
+}
+
+function cssBundler(dirPath) {
+  validateDirPath(dirPath);
+  if (!dirPath.endsWith('/')) {
+    dirPath += '/';
+  }
+  fs.readdir(dirPath, (error, filesFromFS) => {
+    if (error) {
+      console.log(error);
+      throw error;
+    }
+    var cssBundleWriterStream = fs.createWriteStream(dirPath + 'bundle.css');
+    var cssFiles = new Deque(filesFromFS)
+      .filter(file => file.endsWith('.css'))
+      .filter(file => file !== 'bundle.css')
+      .map(fileName => {return dirPath + fileName;});
```

Comment:
In arrow functions if you're handling only return value immediately, you can just put
`.map(fileName => dirPath + fileName);`